indigo-frontend/src/app/pages/experiment/experiment-card/experiment-card.component.scss

```diff
@@ -0,0 +1,208 @@
+.experiment-card {
```

Comment:
Could you please use tailwind css classes as much as possible as we are trying almost not to use css in components.